lib/hawkular.rb

```diff
@@ -50,6 +50,8 @@ def http_get(suburl, headers = {})
     # @param [String] url_part Part of an url to be escaped
     # @return [String] escaped url_part as new string
     def hawk_escape(url_part)
+      return "#{url_part}" if url_part.is_a?(Numeric)
```

Comment:
Why are you wrapping into a string with vars? You could just use `url_part.to_s`, no?